pkg/rules/manager_test.go

```diff
@@ -43,8 +43,14 @@ func (n nopQueryable) Querier(_ context.Context, _, _ int64) (storage.Querier, e
 	return storage.NoopQuerier(), nil
 }
 
+type nopQueryable struct{}
+
+func (n nopQueryable) Querier(_ context.Context, _, _ int64) (storage.Querier, error) {
+	return storage.NoopQuerier(), nil
+}
```

Comment:
@bwplotka  Forgot to add comments here. Please remove `nopQueryable` and `Querier` as well then I guess CI should be happy

Reply:
rebases are hard